nf_core/pipeline-template/assets/methods_description_template.yml

```
@@ -0,0 +1,25 @@
+id: "${workflow.manifest.name.replace('/', '-')}-methods-description"
+description: "Suggested methods description text of pipeline usage"
```

```suggestion
description: "Suggested methods description text of pipeline usage."
```
This solves the duplication issue

---

Whaaaaat? Ok... That's strange. But thank you very much!

---

This smells like a bug. I have a vague feeling that it may have been fixed already? I think it was something to do with the new DOI thing in MultiQC.

---

Ok - I was running with 1.12 when testing so could be.